Mathlib/Analysis/InnerProductSpace/TwoDim.lean

```diff
@@ -489,7 +489,7 @@ theorem kahler_swap (x y : E) : o.kahler x y = conj (o.kahler y x) := by
   have : ∀ r : ℝ, Complex.ofReal' r = @RCLike.ofReal ℂ _ r := fun r => rfl
   simp only [kahler_apply_apply]
   rw [real_inner_comm, areaForm_swap]
-  simp [this]
+  simp [-Complex.coe_algebraMap, this]
```

Comment:
Unfortunately GitHub doesn't let me make a suggestion here, but here's an alternative proof that doesn't rely on going against the `simp` direction:

```lean
theorem kahler_swap (x y : E) : o.kahler x y = conj (o.kahler y x) := by
  simp only [kahler_apply_apply]
  rw [real_inner_comm, areaForm_swap]
  simp [Complex.conj_ofReal]
```

Question for someone who knows more of the analysis side: should we make `Complex.conj_ofReal` a `simp` lemma?